openedx_lti_tool_plugin/models.py

```diff
@@ -274,10 +275,10 @@ def __str__(self) -> str:
         return f'<CourseAccessConfiguration, ID: {self.id}>'
 
 
-class CourseContextManager(models.Manager):
-    """CourseContext Model Manager."""
+class CourseContextQuerySet(models.QuerySet):
```

Comment:
Why did you decided to change this to a queryset?

Reply:
@alexjmpb I changed it to a queryset so I was able to chain the all_for_lti_tool and filter_by_site_orgs querysets, with a Manager I would have to create a Queryset and duplicate the methods of such Queryset in the manager, the as_manager method does that for you automatically: https://docs.djangoproject.com/en/5.0/topics/db/managers/#creating-a-manager-with-queryset-methods